### DISCUSSION TITLE
Maximized window should not take over the whole page

### DISCUSSION THREAD

**Discussion by alaindesilets** [nextapps-de/winbox#92] [Ideas]
When you minimize a window and then click on the maximize icon, the window maximises to the whole size of the web page. 

This is not standard behaviour for a window. Usually, maximize will bring the window back to the dimensions it had before it was minimized.

Note that it's always possible to do that by clicking on the titlebar instead of the maximize icon. But this is not standard UI design and may be confusing to users.

Also, there does not seem to be a way to programmatically bring the window back to its original dimensions (before minimizing). There is the maximize() method, but it too will resize the window to the full size of the page.

Thx for taking this into consideration.

**Comment by ts-thomas** [comment]
> When you minimize a window and then click on the maximize icon, the window maximises to the whole size of the web page.

Yes, because you click on maximize icon. I will think about to change this icon to a "restore" icon when minimized

> Also, there does not seem to be a way to programmatically bring the window back to its original dimensions (before minimizing). There is the maximize() method, but it too will resize the window to the full size of the page.

Just use `.restore()` as described in the readme
